skrf/network.py

```diff
@@ -1133,7 +1131,7 @@ def z0(self):
             return self._z0
 
     @z0.setter
-    def z0(self, z0):
+    def z0(self, z0: Union[Number, npy.ndarray]) -> None:
```

Comment:
could be a list of float/complex as weel as input

Reply:
You are right of course! :)